Compiling with feature simd-stable fails under wasm32-emscripten target

It used to be fine until 0.6.x.
Now it fails with: 
```
error[E0599]: no function or associated item named `zero` found for struct `velocity_constraint_element::VelocityConstraintElement<_>` in the current scope
   --> /home/sugar/.cargo/registry/src/github.com-1ecc6299db9ec823/rapier3d-0.7.2/src/dynamics/solver/velocity_constraint_wide.rs:103:55
    |
103 |                 elements: [VelocityConstraintElement::zero(); MAX_MANIFOLD_POINTS],
    |                                                       ^^^^ function or associated item not found in `velocity_constraint_element::VelocityConstraintElement<_>`
    | 
   ::: /home/sugar/.cargo/registry/src/github.com-1ecc6299db9ec823/rapier3d-0.7.2/src/dynamics/solver/velocity_constraint_element.rs:184:1
    |
184 | pub(crate) struct VelocityConstraintElement<N: SimdRealField> {
    | ------------------------------------------------------------- function or associated item `zero` not found for this
    |
    = help: items from traits can only be used if the trait is implemented and in scope
    = note: the following traits define an item `zero`, perhaps you need to implement one of them:
            candidate #1: `num::Zero`
            candidate #2: `matrixmultiply::kernel::Element`
            candidate #3: `BitBlock`

error[E0599]: no function or associated item named `zero` found for struct `velocity_ground_constraint_element::VelocityGroundConstraintElement<_>` in the current scope
   --> /home/sugar/.cargo/registry/src/github.com-1ecc6299db9ec823/rapier3d-0.7.2/src/dynamics/solver/velocity_ground_constraint_wide.rs:100:61
    |
100 |                 elements: [VelocityGroundConstraintElement::zero(); MAX_MANIFOLD_POINTS],
    |                                                             ^^^^ function or associated item not found in `velocity_ground_constraint_element::VelocityGroundConstraintElement<_>`
    | 
   ::: /home/sugar/.cargo/registry/src/github.com-1ecc6299db9ec823/rapier3d-0.7.2/src/dynamics/solver/velocity_ground_constraint_element.rs:135:1
    |
135 | pub(crate) struct VelocityGroundConstraintElement<N: SimdRealField> {
    | ------------------------------------------------------------------- function or associated item `zero` not found for this
    |
    = help: items from traits can only be used if the trait is implemented and in scope
    = note: the following traits define an item `zero`, perhaps you need to implement one of them:
            candidate #1: `num::Zero`
            candidate #2: `matrixmultiply::kernel::Element`
            candidate #3: `BitBlock`

error: aborting due to 2 previous errors
```

I understand that under emscripten probably simd is not really used, not sure. But given the fact that cargo does not allow features per platform I am forced to disable simd support for all platforms.
 
Since it used to work, maybe this is just a regression?

--- a/.github/workflows/rapier-ci-build.yml
+++ b/.github/workflows/rapier-ci-build.yml
@@ -60,3 +60,18 @@ jobs:
         run: cd build/rapier2d && cargo build --verbose --features wasm-bindgen --target wasm32-unknown-unknown;
       - name: build rapier3d
         run: cd build/rapier3d && cargo build --verbose --features wasm-bindgen --target wasm32-unknown-unknown;
+  build-wasm-emscripten:
+    runs-on: ubuntu-latest
+    env:
+      RUSTFLAGS: -D warnings
+    steps:
+      - uses: actions/checkout@v2
+      - run: rustup target add wasm32-unknown-emscripten
+      - name: build rapier2d
+        run: cd build/rapier2d && cargo build --verbose --target wasm32-unknown-emscripten;
+      - name: build rapier3d
+        run: cd build/rapier3d && cargo build --verbose --target wasm32-unknown-emscripten;
+      - name: build rapier2d --features simd-stable
+        run: cd build/rapier2d && cargo build --verbose --target wasm32-unknown-emscripten --features simd-stable;
+      - name: build rapier3d --features simd-stable
+        run: cd build/rapier3d && cargo build --verbose --target wasm32-unknown-emscripten --features simd-stable;

--- a/src/dynamics/solver/velocity_constraint_element.rs
+++ b/src/dynamics/solver/velocity_constraint_element.rs
@@ -16,7 +16,7 @@ pub(crate) struct VelocityConstraintTangentPart<N: SimdRealField> {
 }
 
 impl<N: SimdRealField> VelocityConstraintTangentPart<N> {
-    #[cfg(not(target_arch = "wasm32"))]
+    #[cfg(any(not(target_arch = "wasm32"), feature = "simd-is-enabled"))]
     fn zero() -> Self {
         Self {
             gcross1: [na::zero(); DIM - 1],
@@ -130,7 +130,7 @@ pub(crate) struct VelocityConstraintNormalPart<N: SimdRealField> {
 }
 
 impl<N: SimdRealField> VelocityConstraintNormalPart<N> {
-    #[cfg(not(target_arch = "wasm32"))]
+    #[cfg(any(not(target_arch = "wasm32"), feature = "simd-is-enabled"))]
     fn zero() -> Self {
         Self {
             gcross1: na::zero(),
@@ -193,7 +193,7 @@ pub(crate) struct VelocityConstraintElement<N: SimdRealField> {
 }
 
 impl<N: SimdRealField> VelocityConstraintElement<N> {
-    #[cfg(not(target_arch = "wasm32"))]
+    #[cfg(any(not(target_arch = "wasm32"), feature = "simd-is-enabled"))]
     pub fn zero() -> Self {
         Self {
             normal_part: VelocityConstraintNormalPart::zero(),

--- a/src/dynamics/solver/velocity_ground_constraint_element.rs
+++ b/src/dynamics/solver/velocity_ground_constraint_element.rs
@@ -15,7 +15,7 @@ pub(crate) struct VelocityGroundConstraintTangentPart<N: SimdRealField> {
 }
 
 impl<N: SimdRealField> VelocityGroundConstraintTangentPart<N> {
-    #[cfg(not(target_arch = "wasm32"))]
+    #[cfg(any(not(target_arch = "wasm32"), feature = "simd-is-enabled"))]
     fn zero() -> Self {
         Self {
             gcross2: [na::zero(); DIM - 1],
@@ -104,7 +104,7 @@ pub(crate) struct VelocityGroundConstraintNormalPart<N: SimdRealField> {
 }
 
 impl<N: SimdRealField> VelocityGroundConstraintNormalPart<N> {
-    #[cfg(not(target_arch = "wasm32"))]
+    #[cfg(any(not(target_arch = "wasm32"), feature = "simd-is-enabled"))]
     fn zero() -> Self {
         Self {
             gcross2: na::zero(),
@@ -143,7 +143,7 @@ pub(crate) struct VelocityGroundConstraintElement<N: SimdRealField> {
 }
 
 impl<N: SimdRealField> VelocityGroundConstraintElement<N> {
-    #[cfg(not(target_arch = "wasm32"))]
+    #[cfg(any(not(target_arch = "wasm32"), feature = "simd-is-enabled"))]
     pub fn zero() -> Self {
         Self {
             normal_part: VelocityGroundConstraintNormalPart::zero(),

